docs/README.md

```diff
@@ -91,12 +101,14 @@ hack/apex_installer.sh
 
 As the project is still in such early development, it is expected that `apex` is run manually on each node you intend to test. If the agent is able to successfully reach the controller API, it will provide a one-time code to provide to the controller web UI to complete enrollment of this node into an Apex Zone.
 
+Note: In a self-signed dev environment, each agent machine needs to have the [imported cert](https://github.com/redhat-et/apex/blob/main/docs/README.md#https) and the [host entry](https://github.com/redhat-et/apex/blob/main/docs/README.md#add-required-dns-entries) detailed above.
```

Comment:
instead of linking to the full github URL, you should be able to do a relative link to the markdown heading